trappy/ftrace.py

```diff
@@ -76,6 +108,22 @@ def __init__(self, name="", normalize_time=True, scope="all",
             setattr(self, attr, trace_class)
             self.trace_classes.append(trace_class)
 
+        cache_path = _trace_cache_path(self.trace_path)
+        cache_good = False
+
+        if _check_trace_cache(self.trace_path):
+            cache_good = True
+            try:
+                # Read csv into frames
+                for trace_class in self.trace_classes:
+                    csv_file = cache_path + '/' + trace_class.unique_word[:-1] + '.csv'
```

Comment:
- `os.path.join`
- I'm not too comfortable with the use of `unique_word[:-1]` - I wonder if we could just us `trace_class.__name__`?